automation/src/test/java/org/greenplum/pxf/automation/features/jdbc/JdbcTest.java

```diff
@@ -34,6 +34,23 @@ public class JdbcTest extends BaseFeature {
             "vc1 varchar(5)",
             "c1 char(3)",
             "bin bytea"};
+    private static final String[] TYPES_TABLE_FIELDS = new String[]{
+            "t1    text",
+            "t2    text",
+            "num1  int",
+            "dub1  double precision",
+            "dec1  numeric",
+            "tm timestamp",
+            "r real",
+            "bg bigint",
+            "b boolean",
+            "tn smallint",
+            "sml smallint",
+            "dt date",
+            "vc1 varchar(5)",
+            "c1 char(3)",
+            "bin bytea",
+            "tmz timestamp with time zone"};
```

Comment:
```suggestion
    private static final String[] TYPES_TABLE_FIELDS = new String[]{
            "t1    text",
            "t2    text",
            "num1  int",
            "dub1  double precision",
            "dec1  numeric",
            "tm    timestamp",
            "r     real",
            "bg    bigint",
            "b     boolean",
            "tn    smallint",
            "sml   smallint",
            "dt    date",
            "vc1   varchar(5)",
            "c1    char(3)",
            "bin   bytea",
            "tmz   timestamp with time zone"};
```

Aligning the column type makes it a little easier to visually parse the column names and types. This is a convention in a lot of the automation tests and while this file doesn't exactly follow it currently, we can follow it with new code.